controllers/telemetry/suite_test.go

```diff
@@ -130,6 +130,10 @@ var _ = BeforeSuite(func() {
 	err = tracepipelineReconciler.SetupWithManager(mgr)
 	Expect(err).ToNot(HaveOccurred())
 
+	telemetryManagerReconciler := NewTelemetryManagerReconciler(client, mgr.GetScheme(), mgr.GetEventRecorderFor("dummy"))
+
```

Comment:
a little nitpick, just to keep the same spacing structure as for the other reconcilers, can you just remove this empty line and have an empty line between `err = telemetryManagerReconciler.SetupWithManager(mgr)` and  `go func() {...}()`?